### ISSUE TITLE
Can not Add Migration for SQLite in UWP application.

### ISSUE THREAD

**Issue by ilya-vrubleuski** [dotnet/efcore#4259]
**I get next error when I execute 'Add-Migration' command via PM**
I followed step by step official [tutorial](http://ef.readthedocs.org/en/latest/getting-started/uwp.html#create-your-database) 

```
System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
  at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
  at System.Reflection.RuntimeAssembly.get_DefinedTypes()
  at Microsoft.Data.Entity.Design.Internal.StartupInvoker..ctor(String startupAssemblyName, String environment)
  at Microsoft.Data.Entity.Design.DbContextOperations..ctor(ILoggerProvider loggerProvider, String assemblyName, String startupAssemblyName, String environment)
  at Microsoft.Data.Entity.Design.MigrationsOperations..ctor(ILoggerProvider loggerProvider, String assemblyName, String startupAssemblyName, String environment, String projectDir, String rootNamespace)
  at Microsoft.Data.Entity.Design.OperationExecutor.<>c__DisplayClass3_0.<.ctor>b__3()
  at Microsoft.Data.Entity.Internal.LazyRef`1.get_Value()
  at Microsoft.Data.Entity.Design.OperationExecutor.<AddMigrationImpl>d__7.MoveNext()
  at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
  at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
  at Microsoft.Data.Entity.Design.OperationExecutor.OperationBase.<>c__DisplayClass4_0`1.<Execute>b__0()
  at Microsoft.Data.Entity.Design.OperationExecutor.OperationBase.Execute(Action action)
```

Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.

**Comment by rowanmiller** [comment]
@Frintezza93 I just went thru that tutorial recently and it was all working. Any chance you could share your project so that we can see what is happening?

**Comment by Frintezza93** [comment]
@rowanmiller I just get the sample from https://github.com/aspnet/EntityFramework.Docs/tree/master/docs/getting-started/uwp/sample.
and change random field. Then I try to execute 'Add-Migration SecondMigration'. And I get the same error.

**Comment by Frintezza93** [comment]
@rowanmiller I`ve tried to add migration on Windows 10 and everything was fine. But now I have Windows 8.1 and I will not be able to update my OS to Windows 10 soon.

**Comment by ErikEJ** [comment]
UWP projects require Windows 10 http://ef.readthedocs.org/en/latest/getting-started/uwp.html

**Comment by Frintezza93** [comment]
@ErikEJ I`ve implemented a few small UWP apps with Windows 8.1 and VS2015. The single problem is adding migration. When I upload application to emulator everything works fine

**Comment by rowanmiller** [comment]
For migrations to work we need to be able to run the application code to get the model. Given you can't run a UWP app on Windows 8.1 there is no way for us to run the code to get the model. Because our design time components require running the application... using an OS where the application can be run is a hard requirement for EF (at least if you want to run migrations). 

As @ErikEJ mentioned, this requirement is included in our documentation.
